package.json

```diff
@@ -18,13 +18,15 @@
   },
   "dependencies": {
     "@slack/web-api": "^5.4.0",
+    "aws": "0.0.3-2",
```

Comment:
What is this?

Reply:
I don't think you want this. Everything you need should be available in `aws-sdk`.

Reply:
I screwed up and installed `aws` not `aws-sdk`